modules/service/common.k

```diff
@@ -30,6 +31,9 @@ schema WorkloadBase(wl.Workload):
     # The number of containers that should be ran.
     replicas?:                   int
 
+    # TopologySpreadConstraint describes how a group of pods ought to spread across topology domains.
+    topologySpreadConstraints:  {str:tp.TopologySpreadConstraint}
```

Comment:
Is this field a must? If not, need to add a question mark.

Reply:
fix.